package gianlucamerlo.exceptions;

public class NotFoundException extends RuntimeException {
    public NotFoundException(long id) {
        super("L'evento con id " + id + " non Ã¨ stato trovato o non esiste ancora!");
    }
}
